src/app/shared/pipes/table-cell-shortener.pipe.ts

```diff
@@ -7,10 +7,16 @@ import { Subscription } from 'rxjs';
   standalone: true,
 })
 export class TableCellShortenerPipe implements PipeTransform {
-  transform(value: string): string {
+  transform(value: string | boolean): string {
     if (value == undefined) {
       return value;
     }
+    if (typeof value === 'boolean' && value === true) {
+      return 'true';
+    }
+    if (typeof value === 'boolean' && value === false) {
+      return 'false';
+    }
```

Comment:
The triple `=` is used to ensure the value and the type is a match. So also checking that the value is a `boolean` is redundant. 

In my head, a lot of bells start ringing when I see a condition asserting a boolean value and then returning a boolean value (albeit a string). This makes the code look rushed and not thought out well.
I'd suggest this to best solution:
```suggestion
    if (typeof value === 'boolean') {
      return value.toString();
    }
```

Because it follows a few simple steps: First, you look at the type that was given, and then you return it with the return type of the method.

The way you did it, it might as well could be done like this:
```suggestion
    if (value === true) {
      return 'true';
    }
    if (value === false) {
      return 'false';
    }
```

Reply:
Yeah you're completely right